src/RenderWebGL.js

```diff
@@ -16,6 +16,13 @@ const log = require('./util/log');
 
 const __isTouchingDrawablesPoint = twgl.v3.create();
 const __candidatesBounds = new Rectangle();
+const __touchingColor = new Uint8ClampedArray(4);
+const __blendColor = new Uint8ClampedArray(4);
+
+// More pixels than this and we give up to the GPU and take the cost of readPixels
+// Width * Height * Number of drawables at location
+const __cpuTouchingColorPixelCount = location.href.includes('count=') ?
+    Number(location.href.match(/count=([^&#]+)/)[1]) : 1000000;
```

Comment:
TODO: Remove this query param - but this should let us play with benchmarks / thresholds:

I.E: http://localhost:8073/playground/index.html?count=1e4#236115215 

anything more than 1000 pixels (width * height * (candidates.length + 1 (ourself) )) would get thrown to GPU